2020-04-18 17:49:37.255 +04:30 [ERR] Coravel task exception: System.Data.SqlClient.SqlException (0x80131904): Timeout expired.  The timeout period elapsed prior to completion of the operation or the server is not responding. ---> System.ComponentModel.Win32Exception (258): The wait operation timed out
   at System.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__122_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteAsync(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.BufferlessMoveNext(DbContext _, Boolean buffer, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncQueryingEnumerable`1.AsyncEnumerator.MoveNext(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.AsyncLinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext(CancellationToken cancellationToken)
   at System.Linq.AsyncEnumerable.Aggregate_[TSource,TAccumulate,TResult](IAsyncEnumerable`1 source, TAccumulate seed, Func`3 accumulator, Func`2 resultSelector, CancellationToken cancellationToken) in D:\a\1\s\Ix.NET\Source\System.Interactive.Async\Aggregate.cs:line 120
   at RealEstateAgency.TaskScheduler.NotifierJob.ProvideUserEvents() in E:\Projects\SolutionsApps\HomeAndShare\WebApi\RealEstateAgency\TaskScheduler\NotifierJob.cs:line 39
   at RealEstateAgency.TaskScheduler.NotifierJob.Invoke() in E:\Projects\SolutionsApps\HomeAndShare\WebApi\RealEstateAgency\TaskScheduler\NotifierJob.cs:line 72
   at Coravel.Scheduling.Schedule.Event.ScheduledEvent.InvokeScheduledEvent()
   at Coravel.Scheduling.Schedule.Scheduler.<>c__DisplayClass22_0.<<InvokeEvent>g__Invoke|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Coravel.Scheduling.Schedule.Scheduler.InvokeEvent(ScheduledEvent scheduledEvent)
ClientConnectionId:4855bbbe-83fa-40cd-9d6d-4b59386f7f1f
Error Number:-2,State:0,Class:11
